code/datums/antagonists/datum_brother.dm

```diff
@@ -0,0 +1,48 @@
+/datum/antagonist/brother
+	name = "Brother"
+	var/special_role = "blood brother"
+	var/datum/objective_team/brother_team/team
+
+/datum/antagonist/brother/New(datum/mind/new_owner, datum/objective_team/brother_team/T)
+	team = T
+	..()
+
+/datum/antagonist/brother/on_gain()
+	SSticker.mode.brothers += owner
+	owner.special_role = special_role
+	owner.objectives += team.objectives
+	finalize_brother()
+	..()
```

Comment:
. = ..() etc

doing it now makes it much easier for someone to make the return value of procs meaningful later on